src/ApplicationManager.cpp

```diff
@@ -174,4 +172,13 @@ void ApplicationManager::notifyTreeStatus(bt::Node::Status status) {
             break;
     }
 }
-}  // namespace rtt
\ No newline at end of file
+
+void ApplicationManager::decidePlay(ai::world::World *world, const ai::world::Field &field) {
+    bool stillValidPlay = playChecker.update(world, field);
+    if (!stillValidPlay) {
```

Comment:
if(!playerChecker.update(world, field)) is shorter. Do not declare variables that are only used once.

Reply:
I think the boolean name is more readable here.